nav2_collision_monitor/src/polygon.cpp

```diff
@@ -562,4 +543,45 @@ inline bool Polygon::isPointInside(const Point & point) const
   return res;
 }
 
+bool Polygon::getPolygonFromString(
+  std::string & poly_string,
+  std::vector<Point> & polygon)
+{
+  std::string error;
+  std::vector<std::vector<float>> vvf = nav2_util::parseVVF(poly_string, error);
+
+  if (error != "") {
+    RCLCPP_ERROR(
+      logger_, "Error parsing polygon parameter: '%s'", error.c_str());
+    RCLCPP_ERROR(
```

Comment:
This should be in the error log above?

Reply:
Bit confused here, are you referring to the error statement or the logger?

Reply:
Ah I see what you mean